components/microsoft_onedrive/actions/download-file/download-file.mjs

```diff
@@ -0,0 +1,48 @@
+import { axios } from "@pipedream/platform";
+import fs from "fs";
+import onedrive from "../../microsoft_onedrive.app.mjs";
+
+export default {
+  name: "Download File",
+  description: "Download a file stored in OneDrive. [See the documentation](https://learn.microsoft.com/en-us/onedrive/developer/rest-api/api/driveitem_get_content?view=odsp-graph-online)",
+  key: "microsoft_onedrive-download-file",
+  version: "0.0.1",
+  type: "action",
+  props: {
+    onedrive,
+    filePath: {
+      type: "string",
+      label: "File Path",
+      description: "The path to the file from the root folder, e.g., `Documents/My Subfolder/File 1.docx`",
```

Comment:
```suggestion
      description: "The path to the file from the root folder, e.g., `Documents/My Subfolder/myFile.docx`",
```

Reply:
This suggestion objective is to avoid using file paths with space

Reply:
The path is URL encoded - why is using spaces a problem?